### ISSUE TITLE
Kucoin - Errors caused by WAX symbol not updated as WAXP

### ISSUE THREAD

**Issue by thetaproject** [CoinAlpha/hummingbot#3521]
**Describe the bug**

When i am trying to create cross exchange market making strategy, i am getting error..

https://user-images.githubusercontent.com/64147266/120937021-16c5b700-c71c-11eb-8cb0-0a5bd4d0573f.mp4

https://user-images.githubusercontent.com/64147266/120937026-19c0a780-c71c-11eb-828d-eecb3488ba68.mp4

please see attached recorded videos, there is very cleare everything..

My goal is to make Kucoin-Maker and Huobi-Taker

when i am creating Huobi-Maker and Kucoin-Taker - everything is good and working.. and when trying to change config file of successed created one, and importing, i am getting Order_amount error.. see screenshot
![hu m2](https://user-images.githubusercontent.com/64147266/120937192-272a6180-c71d-11eb-9631-b44665ac5523.PNG)

[conf_Edited-Kucoin-maker .txt](https://github.com/CoinAlpha/hummingbot/files/6604738/conf_Edited-Kucoin-maker.txt)
[conf_Huobi-Maker .txt](https://github.com/CoinAlpha/hummingbot/files/6604740/conf_Huobi-Maker.txt)

i couldnot upload YML file, so i changed it to txt

log file: 
[logs_hummingbot.log](https://github.com/CoinAlpha/hummingbot/files/6604744/logs_hummingbot.log)

**Comment by RC-13** [comment]
Upon checking, it appears that the Kucoin connector still recognizes the old token symbol for WAX.
![image](https://user-images.githubusercontent.com/57189990/120942198-e7359f80-c759-11eb-9328-1f8941a27278.png)
![image](https://user-images.githubusercontent.com/57189990/120942250-44c9ec00-c75a-11eb-9599-c6f5385a20cc.png)
resulting to the following error when running cross exchange or pure market making strategy using WAX markets:
```python
2021-06-07 05:51:19,370 - 1715 - hummingbot.core.utils.async_utils - ERROR - Unhandled error in background task: list index out of range
Traceback (most recent call last):
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/core/utils/async_utils.py", line 9, in safe_wrapper
    return await c
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/client/command/create_command.py", line 63, in prompt_for_configuration
    await self.prompt_a_config(config)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/client/command/create_command.py", line 106, in prompt_a_config
    prompt = await config.get_prompt()
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/client/config/config_var.py", line 44, in get_prompt
    return await self.prompt()
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/strategy/cross_exchange_market_making/cross_exchange_market_making_config_map.py", line 56, in order_amount_prompt
    min_amount = await minimum_order_amount(maker_exchange, trading_pair)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/client/config/config_helpers.py", line 382, in minimum_order_amount
    mid_price = await get_last_price(exchange, trading_pair)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/core/utils/market_price.py", line 27, in get_last_price
    last_prices = await module.get_last_traded_prices(**args)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/exchange/kucoin/kucoin_api_order_book_data_source.py", line 307, in get_last_traded_prices
    resp_record = [o for o in resp_json["data"]["ticker"] if o["symbolName"] == trading_pair][0]
IndexError: list index out of range
```
The liquidity mining strategy works however it always gets an error:
![image](https://user-images.githubusercontent.com/57189990/120942550-514f4400-c75c-11eb-92c7-f8f53a02da65.png)
```python
2021-06-07 06:39:45,072 - 2685 - hummingbot.connector.parrot - ERROR - Unexpected error while requesting data from Hummingbot API.
Traceback (most recent call last):
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/parrot.py", line 32, in get_campaign_summary
    campaigns = await get_active_campaigns(exchange, trading_pairs)
  File "/Users/clint_g/Documents/Github-Repo/hb-general/hummingbot/connector/parrot.py", line 70, in get_active_campaigns
    for market in campaign_retval["markets"]:
TypeError: string indices must be integers
```

**Comment by thetaproject** [comment]
Hello,

Thank you for reply,

I also was confused about WAX and WAXP..

But i am not so good at scripting and didn't fully understand your reply..
as i understand there is no solution for now yes? This problem must be
fixed with Kucoin API support, yes?

There is Telegram channel about API support for kucoin, and i can write
down there problem,

Maybe you can help me to write down question in right way

Thank you

**Comment by thetaproject** [comment]
![API](https://user-images.githubusercontent.com/64147266/120968195-99c92a80-c779-11eb-9661-7371b806143c.PNG)

**Comment by RC-13** [comment]
@thetaproject Hi, yes for now we don't have a workaround. We don't need to reach out to the KuCoin API support team. My comment is just additional info regarding the issue so that if our development team check into this issue then they would immediately have an idea where to look into.

**Comment by thetaproject** [comment]
Hello,

I can do some donation, if this issue can be fixed in time, if it is
possible... i think this needs not big afford..

Thank you

**Comment by thetaproject** [comment]
Can you tell me waiting for this bug fix in near future or not?

**Comment by RC-13** [comment]
One of our engineers is now looking into this issue although I can't say yet if the fix will be included in our upcoming release this Friday or if it will be on the next release.

**Comment by dennisocana** [comment]
Fixed in version 0.41.0
